### DISCUSSION TITLE
Is this a BUG? , I created a new album and uploaded some pictures, but they cannot be displayed.

### DISCUSSION THREAD

**Discussion by BARERM** [photoprism/photoprism#3404] [Q&A]
![image](https://github.com/photoprism/photoprism/assets/27011525/8fda9ec5-efb4-4319-82be-13db1fa163d0)

**Comment by graciousgrey** [comment]
There could be many reasons why images are missing:

- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

I would recommend to check the errors, logs and review section first.
